salesforce/backend/compiler.py

```diff
@@ -596,6 +601,28 @@ def as_sql(self, qn, connection):
 
 	models.Field.register_lookup(IsNull)
 
+
+	class Range(models.Field.get_lookup(models.Field(), 'range')):
```

Comment:
Not to be pedantic, but I feel like this syntax is a little un-Pythonic. What do you think about doing something like:

```python
BaseRange = models.Field.get_lookup(models.Field(), 'range')
class Range(BaseRange):
    ...
```